note-begin

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 
 initialLocation="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source "${initialLocation}/date-parser"
 
 . ${initialLocation}/note-sync
 
-document_date="${1:-today}"
+# generateDate (function) and generatedDate (variable) are sourced from date-parser
+generateDate $1
```

Comment:
I'm not sure if I should use `"${1}"` here instead, or should it only be used when parsing/reading arguments ?

Reply:
I think it should be fine :+1: